tonic_validate/metrics/latency_metric.py

```diff
@@ -0,0 +1,30 @@
+import logging
+from tonic_validate.classes.llm_response import LLMResponse
+from tonic_validate.metrics.metric import Metric
+from tonic_validate.services.openai_service import OpenAIService
+
+logger = logging.getLogger()
+
+
+class LatencyMetric(Metric):
+    name: str = "latency_metric"
+
+    def __init__(self, target_time: float = 5.0) -> None:
```

Comment:
This default came from looking at how long llama index takes to run rag

Reply:
Did you consider just having the metric report back the response time in milliseconds?  I think reporting it back as a % is not really super intuitive so for folks viewing metrics for the first time it wont make sense whereas a simple latency number would make sense.  What we are essentially doing here is just reporting back how long it takes as a multiple of 5ms which will always require a trip to the documentation to decipher for casual metrics viewers.